server/command.go

```diff
@@ -68,8 +68,11 @@ func (p *Plugin) executeCommand(c *plugin.Context, args *model.CommandArgs) (str
 			return "", nil
 		}
 
-		zoomUser, authErr := p.authenticateAndFetchZoomUser(userID, user.Email, args.ChannelId)
+		zoomUser, authErr := p.authenticateAndFetchZoomUser(user)
 		if authErr != nil {
+			if appErr := p.storeUserState(userID, args.ChannelId); appErr != nil {
```

Comment:
I am 0/5 about this approach. I will leave the next reviewer to give his opinion on this.